kyo-core/shared/src/test/scala/kyo/ChannelTest.scala

```diff
@@ -670,6 +698,7 @@ class ChannelTest extends Test:
         }
 
         "putBatch and takeExactly" in run {
+            val repeats = 1
```

Comment:
Is this intentional?

Reply:
My bad -- leftover from when I was trying to narrow down an issue.